template.go

```diff
@@ -22,18 +22,28 @@ package main
 
 import (
 	"io/ioutil"
+	"path"
 	"time"
 
 	"gopkg.in/yaml.v2"
 )
 
 type template struct {
-	Service string            `yaml:"service"`
-	Thrift  string            `yaml:"thrift"`
-	Method  string            `yaml:"method"`
-	Headers map[string]string `yaml:"headers"`
-	Request interface{}       `yaml:"request"`
-	Timeout time.Duration     `yaml:"timeout"`
+	Peers           []string          `yaml:"peers"`
+	Peer            string            `yaml:"peer"`
+	PeerList        string            `yaml:"peerlist"`
```

Comment:
should we use lower camel which is what most thrift fields will look like in the YAML file?